### PR TITLE
upgrade react-styleguidist-visual

### PR DESCRIPTION
**Pull request by chrismohr**
Upgrades to:
- react-stylguidist 
- react-stylguidist-visual

This resulted in a new version of Chromium coming in. 

While it's good to be on the latest, unfortunately the font rendering has changed slightly and has affected most of our diffs.

I've run many Travis runs to ensure that the tests are stable.

### CODE REVIEW

File: config/styleguide-visual/config.js [hunk 1]
```diff
@@ -12,7 +12,8 @@ module.exports = {
     },
   },
   launchOptions: {
-    args: ['--no-sandbox', '--disable-setuid-sandbox'],
+    args: ['--no-sandbox', '--disable-setuid-sandbox', '--font-render-hinting medium'],
+    slowMo: 40,
```
Comment (chrismohr) [inline config/styleguide-visual/config.js]: Prevents incorrect clipping.

https://github.com/unindented/react-styleguidist-visual/issues/17

File: config/styleguide-visual/config.js [hunk 2]
```diff
@@ -12,7 +12,8 @@ module.exports = {
     },
   },
   launchOptions: {
-    args: ['--no-sandbox', '--disable-setuid-sandbox'],
+    args: ['--no-sandbox', '--disable-setuid-sandbox', '--font-render-hinting medium'],
```
Comment (chrismohr) [inline config/styleguide-visual/config.js]: While I didn't see evidence of this flag helping, I have a slight preference to keep it incase the default changes again.
https://github.com/GoogleChrome/puppeteer/issues/2410

File: package.json [hunk 3]
```diff
@@ -105,6 +105,7 @@
     "test:visual:approve": "styleguidist-visual approve --config config/styleguide-visual/config.js",
     "test:visual": "run-s clean:build test:visual:build test:visual:run",
     "test:visual:component": "run-s clean:build test:visual:build \"test:visual:run -- --filter={1}\" --",
+    "test:visual:component:run": "run-s \"test:visual:run -- --filter={1}\" --",
```
Comment (chrismohr) [inline package.json]: Like `test:visual:run` but for a single component.

File: config/styleguide-visual/config.js (hunk 2 above)
Comment (Mins) [reply config/styleguide-visual/config.js]: Might want to try with the assignment symbol `'--font-render-hinting=medium'`

**Pull request merged**